root/user/UserProfile.js

```diff
@@ -661,12 +672,17 @@ const UserProfileStatistics = ({
         <thead>
           <tr>
             <th colSpan="2">
-              {exp.l('Added entities')}
+              <abbr title={l('Newly applied edits may ' +
+                             'need 24 hours to appear')}
+              >
+                {exp.l('Added entities')}
+              </abbr>
             </th>
           </tr>
         </thead>
         <tbody>
           {Object.keys(ADDED_ENTITIES_TYPES)
+            .filter(type => (type !== 'area' || user.is_location_editor))
```

Comment:
The same should probably be done for instruments?

Reply:
I wouldn't necessarily filter these out in that it's possible that a user used to be a location or relationship editor (hence added areas or instruments) and no longer is. But my only other suggestion would be to hide those if the number is 0. To be fair, we could just not show any line where the number is 0.

Reply:
There's no INSTRUMENT_EDITOR_FLAG right? Anyway, hiding empty rows makes more sense, not displaying entity types that people are not allowed to create (instruments) or maybe not interested in. And this will reduce the height of this table so that it stays closer to the other tables on the page